src/spyglass/common/common_position.py

```diff
@@ -898,17 +899,28 @@ def make(self, key):
             VideoFile()
             & {"nwb_file_name": key["nwb_file_name"], "epoch": epoch}
         ).fetch1()
-        io = pynwb.NWBHDF5IO(raw_dir() + video_info["nwb_file_name"], "r")
+        io = pynwb.NWBHDF5IO(raw_dir + "/" + video_info["nwb_file_name"], "r")
         nwb_file = io.read()
         nwb_video = nwb_file.objects[video_info["video_file_object_id"]]
-        video_filename = nwb_video.external_file.value[0]
+        video_filename = nwb_video.external_file[0]
 
         nwb_base_filename = key["nwb_file_name"].replace(".nwb", "")
         output_video_filename = (
             f"{nwb_base_filename}_{epoch:02d}_"
             f'{key["position_info_param_name"]}.mp4'
         )
 
+        # ensure standardized column names
+        if raw_position_df.columns[0] == "xloc1":
+            columns = ["xloc", "yloc", "xloc2", "yloc2"]
+            raw_position_df.columns = columns
```

Comment:
Should this reuse the existing private method, and turn it into a helper at the bottom of the script?